src/asciidoc/docs/develop/editor.adoc

```diff
@@ -36,30 +36,44 @@ For in-depth walkthroughs of working outside the Koji editor, see <<use-git#>>.
 
 == Previews
 
-=== Inline preview
+You can select the type of preview to display in the preview pane.
+The following types are available.
 
-The Live Preview in the editor is a running preview of your template.
+=== Default
 
-The live preview provides these additional options:
+Selecting *Default* shows a running preview of your template.
 
-* *Editing* and *Preview* modes – Toggles between application *Preview* mode and *Editing* mode, which represents what remixers see when customizing values.
-* *Refresh* – Updates the preview with the latest version of your project code.
-* *Show Logs* – Opens a pane showing the console logs of your project.
-This is useful for debugging runtime errors or incorrect behaviors.
+=== Remix
 
-=== Remote preview
+Selecting *Remix* shows what remixers see when customizing values.
 
-The *Remote* tab enables you to test your template in a new tab or on another device by scanning the QR code.
-You can even share the remote preview link with someone else, and they can see the preview as you develop.
+=== More
 
-[NOTE]
-The remote preview link will only work for as long as your editor session is still active, so it is not suitable for long-term sharing.
+Selecting *More* displays a menu of additional previews.
 
-When your template preview is open in another tab, you still have access to the console logs of the remote device.
-This allows you test your template on different devices and computers while working in the editor, including things like debugging different browsers, operating systems, and form factors.
+image::more.png[More menu]
 
-== Route testing
+* *About* - Displays an About page, if you've enabled the associated entitlement.
 
-The *Routes* tab is used for testing backend routes.
-You can build your request and then click *Send Request* to execute it.
-The *Fetch* tab generates a code snippet for the current request that you can easily add to your application's frontend.
+* *Admin* - Displays an Admin page, if you've enabled the associated entitlement.
+
+* *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+
+* *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+
+* *Sticker* - Displays how your template will look if it's imbedded in another template.
+
+* *Thumbnail* - Displays how your template will look if it's shared on social media.
+
+=== Remote previews
+
+You can preview your template remotely using the following buttons.
```

Comment:
Instead of "remotely", let's describe what we mean by remote previews a little more. These "remote" options are for viewing a preview outside of the editor.